### ISSUE TITLE
Cluster: IOPS in B/s unit

### ISSUE THREAD

**Issue by matejzero** [NetApp/harvest#202]
**Describe the bug**

In cluster dashboard, IOPS are shown as Bytes/s unit.

**Environment**

Harvest version: harvest version 21.05.2-1 (commit ce091de) (build date 2021-06-14T20:31:09+0530) linux/amd64
Command line arguments used: /opt/harvest/bin/harvest restart --config /opt/harvest/harvest.yml
OS: CentOS 7
Install method: yum
ONTAP Version: 9.7
Other:

**To Reproduce**

Open Cluster dashboard and look at IOPs graph.

**Expected behavior**

Unit should be iops/s or none.

**Actual behavior**

![Screenshot 2021-06-21 at 15 08 54](https://user-images.githubusercontent.com/4689252/122767133-9cac3b00-d2a2-11eb-88f9-8cf4ae6e15b2.png)

**Possible solution, workaround, fix**

Change unit to IO ops/sec

**Comment by cgrinds** [comment]
@Hardikl can you take a look - this is similar to #186

**Comment by Hardikl** [comment]
yes, it is same as #186, just it was targeted for node dashboard. 
Found few other graphs in other dashboard also shows wrong unit. I will cover throughput, iops and latency related graphs in all dashboards and handle them here.

**Comment by ruanruijuan** [comment]
Thank you Hardik for being proactive on examining these counters on all dashboards!

**Comment by sridevimm** [comment]
Tested this scenario in Harvest point release 21.05.3.1 and able to reproduce this issue. Cluster IOPs chart still shows unit as kB/s instead of iops/s. Reopening the issue to get this fixed.

<img width="615" alt="Screen Shot 2021-06-23 at 11 55 30 AM" src="https://user-images.githubusercontent.com/83037454/123130408-c2f4e680-d41a-11eb-8d63-c24f5e063441.png">

**Comment by sridevimm** [comment]
Node title shows K io/s. But, the charts are showing millisecs and microsecs. Please take a look at the units for those charts as well. 

<img width="615" alt="Screen Shot 2021-06-23 at 12 03 41 PM" src="https://user-images.githubusercontent.com/83037454/123131471-b8871c80-d41b-11eb-8137-89e2cd02e9e2.png">

**Comment by rahulguptajss** [comment]
@Hardikl do check once if the test setup had latest grafana dashboards

**Comment by Hardikl** [comment]
![image](https://user-images.githubusercontent.com/83282894/123246157-1cd0db80-d503-11eb-8790-19598b39f7e8.png)
  
![image](https://user-images.githubusercontent.com/83282894/123246282-3ffb8b00-d503-11eb-904c-b8a5568c1551.png)
  
I have installed the latest binary in RPM[10.140.132.80 root/netapp], alongwith the grafana(8.0.3) and prometheous(2.28.0).
Then imported the dashboards via this:
$ bin/grafana import --addr my.grafana.server:3000 --directory grafana/prometheus

Cluster and node dashboard graphs are showing as expected. Also I have checked that the changes are available in release branch as well, so moving this to testme.
